service/src/main/java/bio/terra/workspace/app/configuration/spring/WorkspaceTransactionAnnotationConfigurer.java

```diff
@@ -0,0 +1,26 @@
+package bio.terra.workspace.app.configuration.spring;
+
+import bio.terra.workspace.app.configuration.external.WorkspaceDatabaseConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.support.JdbcTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.transaction.annotation.TransactionManagementConfigurer;
+
+@Configuration
+@EnableTransactionManagement
+public class WorkspaceTransactionAnnotationConfigurer implements TransactionManagementConfigurer {
```

Comment:
Does this need to be a separate class for TPS db or something? if so, add a comment/javadoc explaining why

We already have something similar in `WorkspaceDatabaseConfiguration` (a `transactionManager` bean). If that doesn't work any more, we should probably remove the bean definition from that class

Reply:
I added documentation. This does not replace the need for the transaction manager bean. Its purpose is to make the workspace database transaction manager the default for `@Transaction` annotation. It keeps the `@WriteTransaction` and `@ReadTransaction` annotations working.

TPS has to use a vanilla `@Transaction` and explicitly specify its transaction manager.